immersion/kubeflow_pipelines/cicd/solutions/lab-03_vertex.ipynb

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
<p>for the config file location, could we remove the leading "./" for clarity? the form already includes a leading "/".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/117/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I believe this won't work with out making it explicit it is a relative path with the "./" prefix. </p>